modules/star.cfmod.pl

```diff
@@ -45,21 +45,22 @@
 # --mem. Return the required memory allocation.
 if($required_mem){
     if(!$run_fn){
-        print '10G';
+        print '32000000000';
```

Comment:
It doesn't matter, but just so you know - this is parsed by the core `cf` program into bytes, so for increased code readability, outputting `32G` here is fine..